script/reset-environment.sh

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env bash
+if ! [ -x "$(command -v yarn)" ]; then
+    echo "Installing yarn..."
+    npm i -g yarn@0.21.3
+    if [ $? -eq 0 ]; then
+        echo "Yarn successfulling installed globally."
+    fi
```

Comment:
Happy to but it will be in the middle of the npm error output. like this:

> vets-website@1.0.0 reset:env /Users/vacoholmer6/Repos/dsva/vets-website
> ./script/reset-environment.sh

Installing yarn...
npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/vacoholmer6/.nvm/versions/node/v6.10.2/bin/node" "/Users/vacoholmer6/.nvm/versions/node/v6.10.2/bin/npm" "i" "-g" "yarnf@0.21.3"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/yarnf
npm ERR! 404
npm ERR! 404  'yarnf' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/vacoholmer6/Repos/dsva/vets-website/npm-debug.log
Yarn failed to install...please install manually.

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/vacoholmer6/.nvm/versions/node/v6.10.2/bin/node" "/Users/vacoholmer6/.nvm/versions/node/v6.10.2/bin/npm" "run" "reset:env"
npm ERR! node v6.10.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! vets-website@1.0.0 reset:env: `./script/reset-environment.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vets-website@1.0.0 reset:env script './script/reset-environment.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the vets-website package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./script/reset-environment.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs vets-website
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls vets-website
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/vacoholmer6/Repos/dsva/vets-website/npm-debug.log